UnitySDK/Assets/ML-Agents/Scripts/Brain.cs

```diff
@@ -34,6 +40,30 @@ public abstract class Brain : ScriptableObject
         public void SubscribeAgentForDecision(Agent agent)
         {
             LazyInitialize();
+            if (m_SensorShapes.Count == 0)
```

Comment:
Can you put this checking code into a separate method with a fancy name so It is easier to know what it does?

Reply:
Moved to ValidateAgentSensorShapes()